Management/DecisionEngine.cs

```diff
@@ -111,7 +148,7 @@ public async Task<Recommendation> GetStateInfoAsync(CountryCode countryCode, Sta
             var covidScore = await _covidDataClient.CalculateScoreForStateAsync(state, countryCode, cancellationToken);
             var weatherScore = await _weatherDataClient.CalculateScoreForStateAsync(state, countryCode, cancellationToken);
 
-            var finalScore = (airScore.score + covidScore.score + weatherScore.score) / 3;
+            var finalScore = Math.Round((airScore.score + covidScore.score + weatherScore.score) / 3, 1);
```

Comment:
Well this works, but I still think the weighted score system make more sense. As in the real world different factor will be weighted differently

Reply:
We can definitely keep the weighted score system in. :)